packages/console/src/components/Table/TableError.tsx

```diff
@@ -31,7 +31,7 @@ const TableError = ({ title, content, onRetry, columns }: Props) => {
           <div className={styles.content}>
             {content ?? t('admin_console.errors.unknown_server_error')}
           </div>
-          {onRetry && <Button title="general.retry" onClick={onRetry} />}
+          {onRetry && <Button title="admin_console.general.retry" onClick={onRetry} />}
```

Comment:
Hmmm, a common issue in this PR. I think now we all use `admin_console` prefix, am I right? Then we can eliminate the usage of `I18nKey` and use `AdminConsoleKey` instead. Also use `admin_console` as the keyPrefix when calling `useTranslation`.
Then we don't have to add `admin_console` to each phrase any more.

Reply:
agreed

Reply:
This is in another issue: [LOG-3104](https://linear.app/silverhand/issue/LOG-3104)

Reply:
ok